pkg/settings/setting.go

```diff
@@ -261,7 +294,7 @@ func getMetadataConfig() string {
 	return string(ans)
 }
 
-// GetSettingByID returns a setting that is stored with the given id
+// GetSettingByID returns a setting that is stored with the given id.
```

Comment:
nit: some inconsistency with punctuation. I think normally we only added it if it's multiple lines 🤷 .